### PR TITLE
batch write test: add unsupport check test

### PR DESCRIPTION
**Pull request by marsishandsome**

### CODE REVIEW

**Comment by marsishandsome** [comment]
@zhexuany PTAL

File: tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java [hunk 1]
```diff
@@ -155,6 +155,15 @@ TableInfo toProto() {
         .build();
   }
 
+  public boolean hasPrimaryKey() {
+    for (TiColumnInfo col : getColumns()) {
+      if (col.isPrimaryKey()) {
+        return true;
```
Comment (zhexuany) [inline tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java]: maybe we should add a field value?  Once we find a primary key column, we assign the value to the field value.

Comment (marsishandsome) [reply tikv-client/src/main/java/com/pingcap/tikv/meta/TiTableInfo.java]: done

**Comment by marsishandsome** [comment]
@zhexuany PTAL

**Pull request merged**